src/functionalTest/java/uk/gov/hmcts/divorce/testutil/FunctionalTestSuite.java

```diff
@@ -148,6 +148,11 @@ protected Response triggerCallback(Map<String, Object> caseData, String eventId,
     }
 
     protected Response triggerCallback(Map<String, Object> caseData, String eventId, String url, State state) throws IOException {
+        return triggerCallback(caseData, eventId, url, state, state);
```

Comment:
Not 100% sure what this change is for?

Reply:
The original triggerCallback method only accepted a single state which would be applied to the caseDetails and beforeDetails objects.  For the new tests it is necessary to be able to provide independent state values to these objects.  I altered the triggerCallback method to accept the extra state param, then added this overload to negate having rewriting other tests that only need to pass a single state value (which seems to be in line with other overloads elsewhere in the test suite).

Reply:
Removed - no longer required after refactor